src/frontend/src/lib/components/auth/AuthGuard.svelte

```diff
@@ -7,7 +7,7 @@
 {#if $authNotSignedIn}
 	<LandingPage />
 {:else}
-	<div in:fade>
+	<div in:fade class="sm:block sm:overflow-y-scroll">
```

Comment:
I would use `overflow-y-auto` instead of scroll. That way the scrollbar on Mac is only displayed if necessary.

Reply:
changed!